### ISSUE TITLE
blank page for amc once logged in

### ISSUE THREAD

**Issue by stevengonsalvez** [ibm-datapower/appliance-management-center#1]
After following all the steps , https://localhost:8443/amc pops up with log in page. After logging in it is just a blank screen - in all browsers .

**Comment by ghost** [comment]
This sounds like Tomee was started from a different directory and it is having trouble finding the wamc.properties file. Check the contents of the catalina.out file.

If you see error messages about derby, then check if the directories referenced in the wamc.properties file exist (relative to the directory you start Tomee from).

Let me know what you find in the logs so that I can help.

**Comment by tomvanoppens** [comment]
i seem to have the same bug 
this is the error message i get "SEVERE: The property file for the application cannot be loaded. The reported error is 'FileNotFoundException: /home/user/conf/wamc.properties (No such file or directory)'." could this be edited somewhere in the config ?

**Comment by ghost** [comment]
@tomvanoppens Did you do the following step?

copy the files under src/main/tomee/conf to the Apache Tomee conf directory

There is a sample wamc.properties file in there. Just copy both files to the /home/user/conf directory and restart. This should fix the issue. Let me know if it doesn't.

**Comment by ghost** [comment]
I've just committed a small change. Please let me know if that fixes the issue.

**Comment by tomvanoppens** [comment]
no, the issue persists (fresh install, the workaround of copying the conf dir to the referred location still works)
SEVERE: The property file for the application cannot be loaded. The reported error is 'FileNotFoundException: /root/conf/wamc.properties (No such file or directory)'.

**Comment by ghost** [comment]
@tomvanoppens Does the file exist in that location? You must copy the wamc.properties from the source tree into the conf directory under the current directory from where you start the application. It appears you are starting the Tomee from /root, so the code is looking for the wamc.properties file under the path /root/conf as the message implies.

**Comment by tomvanoppens** [comment]
yes, wel it seems it start from the home dir of the user that executes the statup, even if it's  not the start dir , but that's an apache/centos issue

**Comment by ghost** [comment]
I've improved the experience by setting a default values for the wamc.properties file if it is missing. This should avoid the blank page issue.

**Comment by ghost** [comment]
Have you had a chance to try it again?

**Comment by tomvanoppens** [comment]
it's queued, i'll do it when i have the chance (i have to do this on research time when I'm not on a project)
